webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require("path");
+const autoprefixer = require("autoprefixer");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = [
+  {
+    entry: ["./src/index.scss", "./src/index.js"],
+    output: {
+      path: path.resolve(__dirname, "./dist"),
+      filename: "main.js",
+      publicPath: "dist/"
+    },
+
+    module: {
+      rules: [
+        {
+          test: /\.scss$/,
+          use: [
+            {
+              loader: "file-loader",
+              options: {
+                name: "main.css"
+              }
+            },
+            { loader: "extract-loader" },
+            { loader: "css-loader" },
+            {
+              loader: "postcss-loader",
+              options: {
+                plugins: () => [autoprefixer()]
```

Comment:
why did you create postcss.config.js and described the same here? pick either postscss.config.js or this way